library/ix-dev/community/playwright/questions.yaml

```diff
@@ -0,0 +1,367 @@
+groups:
+  - name: Playwright Configuration
+    description: Configure Playwright
+  - name: User and Group Configuration
+    description: Configure User and Group for Playwright
+  - name: Network Configuration
+    description: Configure Network for Playwright
+  - name: Storage Configuration
+    description: Configure Storage for Playwright
+  - name: Resources Configuration
+    description: Configure Resources for Playwright
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+  - variable: playwrightConfig
+    label: ""
+    group: Playwright Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: additionalEnvs
+          label: Additional Environment Variables
+          description: Configure additional environment variables for Playwright.
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: env
+                label: Environment Variable
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: name
+                      label: Name
+                      schema:
+                        type: string
+                        required: true
+                    - variable: value
+                      label: Value
+                      schema:
+                        type: string
+                        required: true
+
+  - variable: playwrightRunAs
+    label: ""
+    group: User and Group Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: user
+          label: User ID
+          description: The user id that Playwright will run as.
+          schema:
+            type: int
+            min: 2
+            default: 568
```

Comment:
Playwright adds a non-root user `pwuser` which is 1000 to the image (https://github.com/microsoft/playwright/blob/main/utils/docker/Dockerfile.jammy#L26).  If running standalone on an empty folder (like for the CI build) then it has to run as 1000, but it runs fine to be configured as the standard `apps` user when running on an actual playwright test project.  So, in my last push I defaulted to 1000 for running but if the user does choose to configure the `playwrightRunAs` it will be 568 by default like other apps.

Please let me know if you would prefer this to be 1000 too though.